packages/babel-plugin-transform-es2015-template-literals/README.md

```diff
@@ -69,22 +68,18 @@ require("babel-core").transform("code", {
 
 `boolean`, defaults to `false`.
 
-In loose mode, tagged template literal objects aren't frozen.
-
-### `spec`
-
-`boolean`, defaults to `false`.
+When `true`, tagged template literal objects aren't frozen. All template literal expressions and quasis are combined with the `+` operator instead of with `String.prototype.concat`.
 
-This option combines all template literal expressions and quasis with `String.prototype.concat`. It will handle cases with `Symbol.toPrimitive` correctly and throw correctly if template literal expression is a `Symbol()`. See [babel/babel#5791](https://github.com/babel/babel/pull/5791).
+When `false` or not set, all template literal expressions and quasis are combined with `String.prototype.concat`. It will handle cases with `Symbol.toPrimitive` correctly and throw correctly if template literal expression is a `Symbol()`. See [babel/babel#5791](https://github.com/babel/babel/pull/5791).
```

Comment:
should "See [babel/babel#5791](https://github.com/babel/babel/pull/5791)." stay, or be removed?